vmdb/spec/models/vmdb_database_connection_spec.rb

```diff
@@ -16,9 +16,10 @@
 
     get_connection = Thread.new do
       VmdbDatabaseConnection.connection.transaction do
-        made_connection.release
-        continue.release
+        # make an empty txn to ensure we've sent something to the db
```

Comment:
I always felt like Rails should just have a `ping` method that does `.execute("SELECT 1 = 1")`